$NetBSD: patch-configure,v 1.6 2025/09/05 06:23:45 adam Exp $

pthread_spin_lock() is portable.
Let pkgsrc figure out the correct paths.

--- configure.orig	2025-05-26 19:27:50.000000000 +0000
+++ configure
@@ -4776,8 +4776,6 @@ main (void)
 
 	      pthread_t pt;
 	      pthread_create(&pt, NULL, NULL, NULL);
-	      pthread_spinlock_t lock;
-	      pthread_spin_lock(&lock);
 
 
   ;
@@ -5651,21 +5649,15 @@ else
       ;;
     freebsd*)
       sysdesc=bsd
-      CPPFLAGS="$CPPFLAGS -I/usr/local/include"
-      LDFLAGS="$LDFLAGS -L/usr/local/lib"
       ;;
     kfreebsd*)
       sysdesc=bsd
       ;;
     netbsd*)
       sysdesc=bsd
-      CPPFLAGS="$CPPFLAGS -I/usr/pkg/include"
-      LDFLAGS="$LDFLAGS -L/usr/pkg/lib -R/usr/pkg/lib"
       ;;
     openbsd*)
       sysdesc=bsd
-      CPPFLAGS="$CPPFLAGS -I/usr/local/include"
-      LDFLAGS="$LDFLAGS -L/usr/local/lib"
       ;;
     dragonfly*)
       sysdesc=bsd
